presto-spi/src/main/java/io/prestosql/spi/type/IntegerType.java

```diff
@@ -18,6 +18,8 @@
 import io.prestosql.spi.block.BlockBuilder;
 import io.prestosql.spi.connector.ConnectorSession;
 
+import java.util.Optional;
```

Comment:
what about `BigintType`, `DecimalType` ?

Reply:
I'll add BigintType since it's easy, but decimal is more involved. I'll do that later as a separate change.